docs/operation/cluster_monitoring.rst

```diff
@@ -0,0 +1,240 @@
+    | PrometheusOutOfOrderTimestamps                  | warning      | Prometheus drops samples with out-of-order timestamps.                                                              |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteStorageFailures                 | warning      | Prometheus fails to send samples to remote storage.                                                                 |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteWriteBehind                     | warning      | Prometheus remote write is behind.                                                                                  |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteWriteDesiredShards              | warning      | Prometheus remote write desired shards calculation wants to run more than configured max shards.                    |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRuleFailures                          | critical     | Prometheus is failing rule evaluations.                                                                             |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusMissingRuleEvaluations                | warning      | Prometheus is missing rule evaluations due to slow rule group evaluation.                                           |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+
+Default Recording rules
+"""""""""""""""""""""""
+
+Recording rules are saved pre-calculated and frequently used time-series
+queries which ensures that MetalK8s administrators no longer have to
+remember complicated expressions which are most of the time computationally
+expensive.
+
+Out-of-the-box, MetalK8s ships with preconfigured recording rules. The
+results of these preconfigured recording rules are saved in the Prometheus
+time series database store.
+
+The contents of :download:`recording rules
+<../../tools/rule_extractor/recording_rule_doc.json>` outlines some of the
```

Comment:
Not sure we want to show these recording rules to the user, not even sure we want to talk about recording rules here.

Reply:
Ok, going to remove this then